tensorboard/plugins/projector/projector_plugin_test.py

```diff
@@ -112,21 +123,26 @@ def testInfoWithValidCheckpointAndEventsData(self):
     self._SetupWSGIApp()
 
     run_json = self._GetJson('/data/plugin/projector/runs')
-    self.assertTrue(run_json)
-    run = run_json[0]
-    info_json = self._GetJson('/data/plugin/projector/info?run=%s' % run)
-    self.assertItemsEqual(info_json['embeddings'], [{
-        'tensorShape': [1, 2],
-        'tensorName': 'var1',
-        'bookmarksPath': 'bookmarks.json'
-    }, {
-        'tensorShape': [10, 10],
-        'tensorName': 'var2'
-    }, {
-        'tensorShape': [100, 100],
-        'tensorName': 'var3'
-    }])
-
+    if USING_REAL_TF:
```

Comment:
I think this test should just be skipped.  The part that checks that the `/runs` endpoint works without real TF is already covered by `testRunsWithValidCheckpoint()` above, and without the events-checking logic it's not worth running this separate test.